sdk/core/core-http/README.md

```diff
@@ -23,14 +23,26 @@ Examples can be found in the `samples` folder.
 
 ## Next steps
 
+First obtain an access token to the management service used in the sample code
+
+One easy way to get the token using Azure CLI (https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
+
+1. `az login` using the above subscription
+2. `az account set -s <subscription id>`
+3. `az account get-access-token --resource=https://management.azure.com`
+
 ### Node.js
 
-- Set the subscriptionId and token
-- Run `node samples/node-sample.js`
+- Set the subscriptionId and token environment variables
+- Change directory to samples folder `cd samples`
```

Comment:
the code sample here feels a bit weird, if we want to provide a code block with the command (and maybe put the text as a comment) that's fine, but the inline thing is odd to me

Reply:
Fixed by putting commands in code blocks